markdown/3.x/en/documentation/upgrade-guide.md

```diff
@@ -447,7 +468,33 @@ tagCloudLink
 ffw-tagCloudLink
 ```
 
+
+### Replacement of identifiers
```

Comment:
I am unsure if we need this section. Weren't those elements meant to be hidden by shadow DOM in the past and hence private? If we don't need it, I think we shouldn't bloat this already huge section.

Reply:
You are right, we don't need it. I hadn't realized the ids were encapsulated in the Shadow DOM until I saw your comment. I checked for version 3.0.0 and yes - they're all hidden from the user.